lib/src/migrators/division.dart

```diff
@@ -0,0 +1,266 @@
+// Copyright 2018 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:args/args.dart';
+
+// The sass package's API is not necessarily stable. It is being imported with
+// the Sass team's explicit knowledge and approval. See
+// https://github.com/sass/dart-sass/issues/236.
+import 'package:sass/src/ast/sass.dart';
+
+import 'package:sass_migrator/src/migration_visitor.dart';
+import 'package:sass_migrator/src/migrator.dart';
+import 'package:sass_migrator/src/patch.dart';
+import 'package:sass_migrator/src/utils.dart';
+
+/// Migrates stylesheets that use the `/` operator for division to use the
+/// `divide` function instead.
+class DivisionMigrator extends Migrator {
+  final name = "division";
+  final description =
+      "Migrates from the / division operator to the divide function";
+
+  @override
+  final argParser = ArgParser()
+    ..addFlag('pessimistic',
+        abbr: 'p',
+        help: "Only migrate / expressions that are unambiguously division.");
+
+  bool get isPessimistic => argResults['pessimistic'] as bool;
+
+  @override
+  Map<Uri, String> migrateFile(Uri entrypoint) =>
+      _DivisionMigrationVisitor(this.isPessimistic, migrateDependencies)
+          .run(entrypoint);
+}
+
+class _DivisionMigrationVisitor extends MigrationVisitor {
+  final bool isPessimistic;
+
+  _DivisionMigrationVisitor(this.isPessimistic, bool migrateDependencies)
+      : super(migrateDependencies: migrateDependencies);
+
+  /// True when division is allowed by the context the current node is in.
+  var _isDivisionAllowed = false;
```

Comment:
I just realized, this should also be set to true in argument lists for functions and mixins, except for the specific case of [new-syntax `rgb()` *et al* functions](https://github.com/sass/sass/blob/master/spec/functions/color.md#rgb-and-rgba). Feel free to fix after landing this PR, though.